tests/migration/migration.go

```diff
@@ -3140,6 +3141,196 @@ var _ = SIGMigrationDescribe("VM Live Migration", decorators.RequiresTwoSchedula
+				secret, err := kubevirt.Client().CoreV1().Secrets(testsuite.GetTestNamespace(secret)).Create(context.Background(), secret, metav1.CreateOptions{})
+				Expect(err).ToNot(HaveOccurred())
+
+				return secretName,
+					"adminredhat",
+					"cat /mnt/user /mnt/password\n",
+					[]libvmi.Option{libvmi.WithSecretFs(secretName, secretName)}
+			}),
+			Entry("DownwardAPI", func() (string, string, string, []libvmi.Option) {
+				downwardAPIName := "downwardapi-" + uuid.NewString()[:6]
+				testLabelKey := "kubevirt.io.testdownwardapi"
+				testLabelVal := "downwardAPIValue"
+
+				return downwardAPIName,
+					testLabelKey + "=" + "\"" + testLabelVal + "\"",
+					"grep --color=never " + testLabelKey + " /mnt/labels\n",
+					[]libvmi.Option{
+						libvmi.WithDownwardAPIFs(downwardAPIName),
+						libvmi.WithLabel(testLabelKey, testLabelVal),
+					}
+			}),
+			Entry("ServiceAccount", func() (string, string, string, []libvmi.Option) {
+				serviceAccountName := "serviceaccount-" + uuid.NewString()[:6]
+
+				return serviceAccountName,
+					testsuite.GetTestNamespace(nil),
+					"cat /mnt/namespace\n",
+					[]libvmi.Option{libvmi.WithServiceAccountFs("default", serviceAccountName)}
+			}),
+		)
+
+		It("should migrate with a shared PVC", func() {
+			_, foundSC := libstorage.GetRWXFileSystemStorageClass()
+
+			if !foundSC {
+				Skip("Skip test when Filesystem RWX storage is not present")
+			}
+
+			pvcName := "pvc-" + uuid.NewString()[:6]
+
+			libstorage.CreateRWXFSPVC(pvcName, testsuite.GetTestNamespace(nil), "1Gi")
+
+			mountVirtiofsCommand := fmt.Sprintf(`#!/bin/bash
+                                      mount -t virtiofs %s /mnt
+									  touch /mnt/test_file
+								`, pvcName)
+
+			vmi := libvmifact.NewFedora(
+				libvmi.WithCloudInitNoCloud(libvmici.WithNoCloudEncodedUserData(mountVirtiofsCommand)),
+				libvmi.WithFilesystemPVC(pvcName),
+				libvmi.WithInterface(libvmi.InterfaceDeviceWithMasqueradeBinding()),
+				libvmi.WithNetwork(v1.DefaultPodNetwork()),
+			)
+
+			vmi = libvmops.RunVMIAndExpectLaunch(vmi, 180)
+
+			By("Logging into the VMI")
+			Expect(console.LoginToFedora(vmi)).To(Succeed())
+
+			By("Starting the migration")
+			migration := libmigration.New(vmi.Name, vmi.Namespace)
+			migration = libmigration.RunMigrationAndExpectToCompleteWithDefaultTimeout(virtClient, migration)
+
+			libmigration.ConfirmVMIPostMigration(virtClient, vmi, migration)
+
+			By("Checking the PVC is still mounted and testing file is accessible")
+			Expect(console.SafeExpectBatch(vmi, []expect.Batcher{
+				&expect.BSnd{S: "test -f /mnt/test_file \n"},
+				&expect.BExp{R: console.PromptExpression},
+				&expect.BSnd{S: "echo $?\n"},
+				&expect.BExp{R: console.RetValue("0")},
+			}, 200)).To(Succeed())
+		})
+
+		It("should migrate with a shared DV", func() {
+			sc, foundSC := libstorage.GetRWXFileSystemStorageClass()
+
+			if !foundSC {
```

Comment:
Please don't use skip anymore but instead the proper decorator

Reply:
Still applies